src/lib/system-info.ts

```diff
@@ -77,7 +77,7 @@ export async function getCpuId(): Promise<string | undefined> {
 	}
 }
 
-const undervoltageRegex = /under.*voltage/;
+const undervoltageRegex = new RegExp('under.*voltage', 'i');
```

Comment:
Whilst the current version will work I normally see this as:
```suggestion
const undervoltageRegex =/under.*voltage/i;
```
With `new RegExp(...)` resolved for when the regexp is dynamically created, eg via string concatenation